.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
not sure what this is doing, but I think it shouldn't be there.

Reply:
yamllint [yells when this is absent](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_start)

Reply:
I'll merge and check what github thinks, it definitely doesn't have to be there.